backend/prisma/migrations/20240714090330_init_restart/migration.sql

```diff
@@ -0,0 +1,8 @@
+-- CreateTable
+CREATE TABLE "Association" (
+    "id" SERIAL NOT NULL,
+    "nerf" JSONB NOT NULL,
+    "buff" JSONB NOT NULL,
```

Comment:
@Kam227 why is this structured this way? What does an example data point in this table look like? Why are the columns called "nerf" and "buff"

Reply:
I think this is similar to my other comments and that this should be structured as rows rather than jsonb